mmpose/core/visualization/image.py

```diff
@@ -176,7 +176,10 @@ def imshow_keypoints_3d(
         ax_img.imshow(img, aspect='equal')
 
     for idx, res in enumerate(pose_result):
-        kpts = res['keypoints_3d']
+        dummy = len(res) == 0
```

Comment:
Currently, we control the number of visualized instances outside this function, and pass empty pose results as a placeholder, right? 

I suggest we add a function argument like "fix_instance_num" here, and pad or truncate the original pose results during visualization. Do you think it would be more clear?

Reply:
Yes, I'll modify the visualization function.